src/components/skipNavLink.js

```diff
@@ -0,0 +1,42 @@
+import { div, h } from 'react-hyperscript-helpers'
+import { Link } from 'src/components/common'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+
+
+export const SkipNavLink = Utils.forwardRefWithName('SkipNavLink', (props, ref) => {
+  return h(Link, {
+    as: 'a',
+    href: '#',
+    className: 'reveal-on-focus',
+    style: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      background: 'white',
+      color: colors.accent(),
+      border: '1px solid black',
+      borderColor: colors.accent(),
+      padding: '1em',
+      boxShadow: 'rgba(0, 0, 0, 0.5) 0px 0px 4px 0px',
+      zIndex: 9998
+    },
+    onClick: event => {
+      event.preventDefault()
+      event.stopPropagation()
+      ref.current?.focus()
+    },
+    ...props
```

Comment:
`_.merge` here to allow merging of styles?

Reply:
I added `_.merge` to the `SkipNavLink` component. For some reason it wasn't working properly in the `SkipNavTarget` component, probably because of the `ref`, so I left that one as-is.